package.json

```diff
@@ -64,6 +63,7 @@
     "react-native-confirmation-code-field": "^7.3.2",
     "react-native-date-picker": "~4.3.3",
     "react-native-gesture-handler": "~2.12.0",
+    "react-native-get-random-values": "~1.9.0",
```

Comment:
I don't see it being used anywhere

Reply:
It's from the docs 🤷‍♂️ https://docs.amplify.aws/react-native/build-a-backend/auth/set-up-auth/#install-amplify-libraries